public/app/features/dashboard/components/SubMenu/DashboardLinksDashboard.tsx

```diff
@@ -21,13 +23,17 @@ export const DashboardLinksDashboard: React.FC<Props> = props => {
   let resolvedLinks = useResolvedLinks(props, opened);
 
   // TODO: PMM-7736 remove it ASAP after migration transition period is finished
```

Comment:
is this TODO still relevant?

Reply:
Yes. I talked with Roma ([here](https://perconacorp.slack.com/archives/C01JNP68V18/p1620937877328000)) and for now we'll keep it